src/g.cpp

```diff
@@ -1158,25 +1206,22 @@ g2 g2::frobeniusMap(int64_t power) const
         r.x.frobeniusMapAssign(1);
         r.y.frobeniusMapAssign(1);
         r.z.frobeniusMapAssign(1);
-        r.x = r.x.mul(frb0);
-        r.y = r.y.mul(frb1);
+        r.x = r.x.multiply(frb0);
+        r.y = r.y.multiply(frb1);
     }
     return r;
 }
 
-// MultiExp calculates multi exponentiation. Given pairs of G2 point and scalar values
+// WeightedSum calculates multi exponentiation. Given pairs of G2 point and scalar values
 // (P_0, e_0), (P_1, e_1), ... (P_n, e_n) calculates r = e_0 * P_0 + e_1 * P_1 + ... + e_n * P_n
 // Length of points and scalars are expected to be equal, otherwise NONE is returned.
```

Comment:
Replace

```
Length of points and scalars are expected to be equal, otherwise NONE is returned.
```

with

```
If length of points and scalars are not the same, 
then missing points will be treated as the zero point 
and missing scalars will be treated as the zero scalar.
```

Reply:
fixed